xslt/styles.xslt

```diff
@@ -143,7 +144,7 @@
         <xsl:attribute name="font-size">13pt</xsl:attribute></xsl:attribute-set>
     <xsl:attribute-set name="h3" use-attribute-sets="bold h-title">
         <xsl:attribute name="font-size">12pt</xsl:attribute></xsl:attribute-set>
-    <xsl:attribute-set name="h4" use-attribute-sets="bold h-title"></xsl:attribute-set><xsl:attribute-set name="h5" use-attribute-sets="italic h-title"></xsl:attribute-set>
+    <xsl:attribute-set name="h4" use-attribute-sets="bold h-title"></xsl:attribute-set>
```

Comment:
Would you mind making sure that this PR only contains changes relevant for this added functionality ? Atomic PR requests per feature / fix / ... make it easier to review, git-blame, ... . Thanks!

Reply:
Yeah of course, should I have two PR's one for the labels and one for the "Accept" status essentially?

Reply:
Yes please, as atomic as possible, one branch / PR per feature / .. .

Reply:
Thanks a bunch in advance!

Reply:
Of course, I made a new PR which has only the labels/multiple table creation (this one is kind of a 2 in one hope it's ok)